### DISCUSSION TITLE
Set max_locks_per_transaction on postgres docker image in github actions workflow file

### DISCUSSION THREAD

**Discussion by toruncem** [actions/virtual-environments#7656] [General questions]
Hello! I was wondering what the right away to configure `max_locks_per_transaction` this is in a github actions workflow file given I am using a docker postgres image

Current my yaml file looks something like this 

>  jobs:
  pytest:
    runs-on:
      group: 'Default Larger Runners'
    services:
      db:
        image: postgres:14
        env:
          POSTGRES_USER: x
          POSTGRES_DB: x
          POSTGRES_PASSWORD: x
        ports:
        - 5432
    - uses: actions/checkout@v3
    - name: Setup Python
      id: setup_python
      uses: actions/setup-python@v4
      with:
        python-version: '3.9'

then come other configs then eventually I run pytest with 

>    name: Run pytest check
      run: PYTHONPATH=. venv/bin/python3 -m pytest -v . -n auto
      env:
        ENV: "TESTING"
        HOST: "0.0.0.0"
        DB_HOST: localhost
        DB_PORT: ${{job.services.db.ports['5432']}}
        DB_BASE: x
        DB_USER: x
        DB_PASS: x

But I'm not sure how to set `max_locks_per_transaction` in this file such that I can avoid an error during pytest in Github Actions due to this error:

PG::OutOfMemory: ERROR: out of shared memory
HINT: You might need to increase max_locks_per_transaction.

This solution didn't work for me: https://github.com/actions/runner-images/issues/527

**Comment by ilia-shipitsin** [comment]
I'd suggest to ask in https://github.com/docker-library/postgres

**Reply by toruncem** [reply]
I think there's a way to do it in a docker file, but I specifically want to do in my github actions workflow file, is there not a way?

**Reply by ilia-shipitsin** [reply]
you refer to https://github.com/actions/runner-images/issues/527
it is not relevant way to manipulate postgres in docker because docker does not have it's own "systemctl"

what is relevant way to set postgres variable in official postgres docker image - I think it is the question to be asked in official postgresql docker tracker

**Reply by toruncem** [reply]
okay, thank you Ilia - I will try that.
